wagtailio/project_styleguide/templates/patterns/includes/header.html

```diff
@@ -5,14 +5,23 @@
         </svg>
     </a>
 
-    {% if get_started_menu_items %}
-        <button class="header__button header__button--get-started button" data-get-started>
-            Get Started
-            <svg class="modal__close-icon" aria-hidden="true">
-                <use xlink:href="#chevron"></use>
+    <div class="header__items-wrap">
+        <button class="header__button header__button--search" aria-label="Open the search" data-site-search>
+            <svg class="header__search-icon" aria-hidden="true">
+                <use xlink:href="#magnifying-glass"></use>
             </svg>
         </button>
-    {% endif %}
+
+        {% if get_started_menu_items %}
+            <button class="header__button header__button--get-started button" data-get-started>
+                Get Started
+                <svg class="modal__close-icon" aria-hidden="true">
```

Comment:
Comment: Does aria-hidden make sense here? The MDN docs [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#description), the quote of ` The aria-hidden attribute can be used to hide non-interactive content from the accessibility API.` makes me think like we don't want it as we can interact with this and the close menu.

Reply:
In this instance I see the icon as decorative so it doesn't need to be in accessibility tree - the screen reader will pick up the button text here. From the same article from MDN it sound like this will benefit users of assistive tech:

> This can improve the experience for assistive technology users by hiding:
> - Purely decorative content, such as icons or images